### PR TITLE
tsunami: new port in audio

### PR DESCRIPTION
**Pull request by barracuda156**
#### Description

New port in audio

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 14.2.1
Xcode 15.1

macOS 10.6
Xcode 3.2

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?
- [ ] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

File: audio/tsunami/Portfile [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               cmake 1.1
+PortGroup               github 1.0
+
+github.setup            momentarylapse tsunami 23a4621c7f2d812e06f0cf6d3ac9553421666ccf
```
Comment (reneeotten) [inline audio/tsunami/Portfile:7]: as usual, please do not use git hashes instead you should use a released tag used by upstream.

**Review by reneeotten — CHANGES_REQUESTED** [review]

File: audio/tsunami/Portfile (hunk 1 above)
Comment (barracuda156) [reply audio/tsunami/Portfile:7]: @reneeotten This is not possible, release is broken for macOS. Please follow the issue I refer: https://github.com/momentarylapse/tsunami/issues/12

Comment (pmetzger) [reply audio/tsunami/Portfile:7]: Do we have a resolution on this?

Comment (barracuda156) [reply audio/tsunami/Portfile:7]: Well I do not see a problem to switch to release once it is there. I do not pick commits just for fun or to annoy someone :) And I do move to released versions once it works, I did that with other ports.

**Comment by kencu** [comment]
this is not carried by any distro, and has hardly any forks or users.

https://repology.org/projects/?search=tsunami&maintainer=&category=&inrepo=&notinrepo=&repos=&families=&repos_newest=&families_newest=

It has no working release, and we have no confirmation that it actually works to edit audio on macos systems, or which systems it might work to edit audio on, if it does work. The author says he doubts it can work on arm in the referenced discussion.

And I guess it raises the question just where the bar might be set for something to be officially added to macports.

**Comment by barracuda156** [comment]
@kencu I just forgot that all the rest of Macports robustly works on arm.

**Comment by kencu** [comment]
pretty much does, yes.

**Comment by barracuda156** [comment]
> pretty much does, yes.

In reality numerous port do not even _build_ on arm64, forget working.

**Comment by kencu** [comment]
**In reality**, point them out with a ticket, they will get fixed if so. 

Some things are so absolutely ancient and nearly useless (qt4, for example) that they are not worth spending time on and they just run in emulation. Although even there, if there is any point to the port at all, the build will get fixed for arm.

**Pull request closed without merging**